accepted/2021/system-drawing-win-only/system-drawing-win-only.md

```diff
@@ -0,0 +1,193 @@
+# Make `System.Drawing.Common` only supported on Windows
+
+**Owner** [Santiago Fernandez Madero](https://github.com/safern)
+
+In the .NET Core 2.x days we created the `Microsoft.Windows.Compatibility`
+package which exposes many Windows-only APIs so that more people can migrate to
+.NET Core. As part of this, we also ported `System.Drawing.Common`. This was
+mainly meant for .NET Core users that still target Windows, however since Mono
+already had a cross-platform implementation, we decided to bring that over. The
+cross-platform functionality is provided via the native library `libgdiplus`.
+Sadly, `libgdiplus` is a community supported library and at this point
+effectively unmaintained.
+
+While `libgdiplus` has helped a lot of customers expand to cross-platform it has
+also caused some headaches. `System.Drawing.Common` is very Windows driven as it
+is mostly a wrapper around GDI+ and is heavily coupled to Windows Forms, which
+keeps evolving and diverging from cross-platform graphics. This has caused some
+confusion over why some things don't work well on Unix or don't work at all.
+
+Since the inclusion of the Mono cross-platform implementation we spent lot of
+time redirecting issues to `libgdiplus` that never got fixed, or helping people
+install `libgdiplus` correctly (because it's not part of .NET Core proper). We
+have thought about our options, and to support `System.Drawing.Common` properly
+on Unix we'd need to make `libgdiplus` a first-class citizen and ship it as part
+of the product:
+
+* `libgdiplus` doesn't ship with all operating systems by default, just like
+  other native libraries we depend on (icu, openssl).
+
+* `libgdiplus` is community driven, so a lot of issues/PRs just go stale and are
+  never looked at.
+
+* There is no servicing story for bugs and security issues.
+
+However, making `libgdiplus` a first-class citizen is costly. It's around 30k
+lines of pretty old C code, virtually untested, and lacks a lot of
+functionality. `libgdiplus` also has a lot of external dependencies for all the
+image processing and text rendering, such as `cairo`, `pango`, and other native
```

Comment:
Do we know how actively supported (and widely distributed) those dependencies are too? All the same requirements apply to them.

Reply:
@safern ?

Reply:
Cairo is a core library to GTK still, I believe. I think that's also the case with Pango but I'm not sure.

Reply:
Yes cairo is actively supported and widely distributed on Debian and Fedora distros. For macOS you still need to install it manually.

And yes, pango is `GTK` still.